web/profiles/custom/os2loop/modules/os2loop_search_db/src/Helper/Helper.php

```diff
@@ -174,4 +184,28 @@ public function getContentTypeGroups(): array {
       ];
   }
 
+  /**
+   * Implements hook_form_alter().
+   */
+  public function formAlter(array &$form, FormStateInterface $form_state, $form_id) {
+    if ('views_exposed_form' === $form_id) {
```

Comment:
Do we want this on all exposed forms?

Reply:
I've strengthened the check for when to do this in https://github.com/itk-dev/os2loop/pull/142/commits/9a352011585e986bbf3bd32d502f6ebfda2b58fb.